san/transform.py

```diff
@@ -35,4 +36,11 @@ def eth_top_transactions_transform(data):
         column['toAddressIsExchange'] = column['toAddress']['isExchange']
         column['fromAddress'] = column['fromAddress']['address']
         column['toAddress'] = column['toAddress']['address']
+
+
+def news(data):
```

Comment:
`news_transform` otherwise it won't be called

Reply:
Also add documentation to this module on how it works - that you need to define `<query>_transform` function

Reply:
@IvanIvanoff Just saw the second comment. The documentation should be about the news_transform or the transform functions in general ?

Reply:
About the whole module